pkg/jobs/rbac/rbac.go

```diff
@@ -168,6 +254,33 @@ func ApplyRBAC(kubeset kubernetes.Interface, namespace string) error {
 	return nil
 }
 
+func ApplyRBACHypershift(kubeset kubernetes.Interface, namespace string, curatorNamespace string) error {
+	klog.V(2).Info("Check if RoleBinding curator exists in namespace " + namespace)
+	if _, err := kubeset.RbacV1().RoleBindings(namespace).Get(
+		context.TODO(), "curator", v1.GetOptions{}); k8serrors.IsNotFound(err) {
+		klog.V(2).Info(" Creating RoleBinding curator in namespace " + namespace)
+		_, err = kubeset.RbacV1().RoleBindings(namespace).Create(
+			context.TODO(), getRoleBinding(curatorNamespace), v1.CreateOptions{})
+		if err != nil {
+			return err
+		}
+		klog.V(0).Info(" Created RoleBinding in cluster namespace ✓")
+	}
+
+	klog.V(2).Info("Check if ClusterRole curator-crb exists")
+	if _, err := kubeset.RbacV1().ClusterRoleBindings().Get(
+		context.TODO(), "curator-crb", v1.GetOptions{}); k8serrors.IsNotFound(err) {
+		klog.V(2).Info(" Creating ClusterRoleBinding curator-crb")
+		_, err = kubeset.RbacV1().ClusterRoleBindings().Create(
+			context.TODO(), getClusterRoleBinding(curatorNamespace), v1.CreateOptions{})
+		if err != nil {
+			return err
+		}
+		klog.V(0).Info(" Created ClusterRoleBinding ✓")
+	}
+	return nil
```

Comment:
```suggestion
	return err
```
If I am reading this correctly.